app/_src/gateway-operator/get-started/konnect/horizontal-autoscaling.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Is there a good reason for autoscaling to live under the "Konnect Quick Start"?

Thoughts from my side:
1. HPA doesn't belong in the "quick" zero-to-one path
2. This is now a feature for the Konnect mode only, but this reality will almost certainly change in a future version
3. There are concepts in autoscaling that go beyond a "quick start" for somebody running Konnect :
    1. In this doc: the load test
    2. In general / in a broader sense: tweaks, best practices, interactions with other mechanisms (such as B/G rollouts), explanation of what modes support HPA and which don't

Suggestion: leave the quickstart "quick" and put the HPA documentation in the "production use" cookbook with targeted recipes for particular features - under "Production Deployment".

Reply:
👍 Re: moving this outside of quickstart, makes sense as that's not really a topic one would like to read when trying to figure out how to start using KGO.